docs/framework/configure-apps/redirect-assembly-versions.md

```diff
@@ -157,7 +157,7 @@ You can enable automatic binding redirection if your app targets older versions
 - [Programming with Assemblies](../../../docs/framework/app-domains/programming-with-assemblies.md)
 - [How the Runtime Locates Assemblies](../../../docs/framework/deployment/how-the-runtime-locates-assemblies.md)
 - [Configuring Apps](../../../docs/framework/configure-apps/index.md)
-- [Configuring .NET Framework Apps](http://msdn.microsoft.com/library/d789b592-fcb5-4e3d-8ac9-e0299adaaa42)
+- [Configuring .NET Framework Apps](https://msdn.microsoft.com/library/d789b592-fcb5-4e3d-8ac9-e0299adaaa42)
```

Comment:
The updated URL for the configuration documentation is <https://docs.microsoft.com/en-us/dotnet/framework/configure-apps/>.

Reply:
The updated URL is the same as that one in `Configuring Apps`.